dockstore/dockstore
# Fix description update for existing workflow version

**Description**
This PR fixes the bug where the version's description isn't updated when the description source changes for an existing version (ex: from custom README path to descriptor description, or descriptor description to repo root README).

**Review Instructions**
- Create a GitHub app workflow that has a repo root README, a description in the primary descriptor, and a custom README path in the .dockstore.yml
- Verify that the description displayed is from the custom README path
- Remove the custom README path in the .dockstore.yml
- Verify that the description displayed is from the description in the primary descriptor
- Delete the description in the primary descriptor
- Verify that the description displayed is from the repo root README

**Issue**
#5456 
https://ucsc-cgl.atlassian.net/browse/DOCK-2370

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!
